Menu homologado

conaaxo opened the pull request.

Commit:
Menu homologado

conaaxo committed:
Menu Homologado

Menu Homologado 18 Abril

conaaxo committed:
Add files via upload

conaaxo committed:
Menu Homologado

Menu Homologado 18 Abril

conaaxo committed:
Delete base-axo.html

conaaxo committed:
Menu Circulo

Menu Circulo

conaaxo committed:
Menu Homologado

conaaxo committed:
Homologar Menú

conaaxo committed:
Menú homologado

conaaxo committed:
Titles, Author & Description

conaaxo committed:
Actualizaciones Menú

conaaxo committed:
LIGHTBOX LA GRAN FAMILIA

conaaxo committed:
Update comollegan.css

albertoglezba commented on assets/img/conoce/vecinosverde/bottom.png:
Es preferible establecer fondo con CSS en vez de con imagen

conaaxo replied:
listo

conaaxo committed:
ENCABEZADO PUBLICACIONES

conaaxo committed:
LA GRAN FAMILIA

albertoglezba commented on arboles.html:
```diff
@@ -1,162 +1,66 @@
+    <script src="js/jquery-2.2.4.min.js"></script>
+
+    <!-- Font Awesome icons (free version)-->
+    <script src="https://use.fontawesome.com/releases/v5.15.4/js/all.js" crossorigin="anonymous"></script>
+    <!-- Google fonts-->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Pangolin&family=Work+Sans:ital,wght@0,300;0,400;0,800;1,400&display=swap"
+          rel="stylesheet">
```
En un futuro no muy lejano hay que revisar cual de todas estas fuentes se utilizan y cuales no...
Considero que sí son pocos los iconos que se utilizan de fontAwesome, sería conveniente mejor bajarlos de fontello ya que el tamaño de la fuente, js y css disminuye bastante

conaaxo replied:
Se considera para la siguiente versión

conaaxo replied:
fontawesome.com lo dejo solo en las páginas necesarias por el momento

albertoglezba commented on assets/img/actividades/colorear/patronC.png:
También este patrón puede ser generado con CSS

conaaxo replied:
ok

conaaxo replied:
Listo

albertoglezba commented on assets/mega-site-navigation/css/style.css:
```diff
@@ -104,15 +104,15 @@ Main components
 
 .cd-main-content {
-  background: black;
-  min-height: 100vh;
+  /*background: black;*/
+  /*min-height: 100vh;*/
```
como aún no contamos con un minificador en producción, lo mejor es no subir al PR lineas de código comentado

conaaxo replied:
ok, reduzco mis comentarios,

albertoglezba commented on cantos.html:
Borrar código comentado

albertoglezba commented on carteles.html:
```diff
@@ -93,6 +94,16 @@ <h3>Carteles</h3>
                                             ejemplares para escuelas de educación indígena de la SEP.<br/>
                                             500,000 en Enero de 2014
                                         </p>
+                                        <br>
+                                        <br>
+                                        <br>
+                                        <br>
+                                        <br>
+                                        <br>
+                                        <br>
+                                        <br>
+                                        <br>
+                                        <br>
```
Usar CSS para dar márgenes, en este caso las clases auxiliares de bootstrap quedaría mejor

conaaxo replied:
claro

conaaxo replied:
era prueba, pero se solucionó con CSS

albertoglezba commented on css/color.css:
```diff
@@ -496,4 +495,20 @@ body {
+  background-image: url(../assets/img/actividades/colorear/titocolorear.png);
+}
+
+footer{
+  position: fixed !important; /* AXO */
+  bottom: 0;
```
En pantallas pequeñaas o tabletas apaisadas  no se aprecian bien los iconos de las demás actividades para colorear, sugiero que el footer no sea fixed, y en todo caso usas las clases auxiliares de bootstrap para tal efecto

conaaxo replied:
se trabaja la adaptabilidad para todas las actividades en la siguiente versión

albertoglezba commented on color_golfo.html:
```diff
@@ -1,26 +1,51 @@
-    <!-- InstanceEndEditable -->
-    <script src="js/jquery-1.11.1.min.js"></script>
+
     <script src="js/jquery.stellar.min.js"></script>
     <script src="js/jquery.sticky-kit.min.js"></script>
     <script src="js/waypoints.min.js"></script>
```
¿Se utilizan aquí estos plugins?

conaaxo replied:
ya lo probé en las secciones de colorear y no veo falla, los quito

albertoglezba commented on color_matorral.html:
```diff
@@ -242,21 +153,40 @@ <h1><a href="index.html">M&eacute;xico el pa&iacute;s de las maravillas</a></h1>
+<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js"></script>
+<!-- Core theme JS-->
 
+<!--wow js-->
+<script src="js/wow.js"></script>
+<!--scripts fuera del template base, tomé los del morphing modals, pero podrían ser en una carpeta afuera-->
```
Tenía entendido wow.js se ocupa solo en el index ¿es correcto?

conaaxo replied:
Ya lo eliminé de algunas secciones y no veo errores, lo voy removiendo

albertoglezba commented on color_pacifico.html:
mismas recomendaciones que los anteriores de los archivos de colorear

albertoglezba commented on color_pastizal.html:
mismas recomendaciones que los anteriores de los archivos de colorear

albertoglezba commented on color_selvah.html:
mismas recomendaciones que los anteriores de los archivos de colorear

albertoglezba commented on colorear.html:
```diff
@@ -0,0 +1,195 @@
+                <a href="http://www.biodiversidad.gob.mx/" target="_blank"></a>
+            </div>
+        </div>-->
+        <!--banderín BIO-->
```
evitar subir código comentado

albertoglezba commented on colorear.html:
```diff
@@ -0,0 +1,195 @@
+    <link rel="icon" type="image/x-icon" href="assets/img/favicon.png"/>
+
+    <!-- jquery del CDN de google -->
+    <script src="js/jquery-2.2.4.min.js"></script>
```
jquery duplicado, ya hay una versión 2.1x abajo

albertoglezba commented on colorear.html:
```diff
@@ -0,0 +1,195 @@
+<!--wow js-->
+<script src="js/wow.js"></script>
+<!--scripts fuera del template base, tomé los del morphing modals, pero podrían ser en una carpeta afuera-->
+<script src="assets/morphing-modal/js/jquery-2.1.1.js"></script> <!-- jquery -->
```
Esta versión de jQuery es la que se tendría que eliminar

albertoglezba commented on css/ahorcado.css:
```diff
@@ -340,4 +340,9 @@ body {
   display: inline-block;
+}
+
+footer {
+  position: fixed !important;
+  bottom: 0;
```
Misma recomendación que en la sección de colorear, para evitar reglas de css duplicadas, utilizar clases de bootstrap (o evitar que el footer quede fijo)

conaaxo replied:
se trabaja menú y nuevo  footer para las actividades de la siguiente versión

albertoglezba commented on css/cuidas.css:
```diff
@@ -304,4 +310,40 @@ footer{
+
+@media only screen and (min-width: 1170px) {
+    #instActividad p {
+      font-size: 20px;
```
para tamaños de fuente en distintos dispositivos quizá convenga más usar `rem` en vez de `px`

albertoglezba commented on css/cuidas_tablet.css:
¿Es necesario este archivo para tabletas? 
¿Se podría realizar con marcado de Bootstrap?

conaaxo replied:
ya se eliminan varios tablet css innecesarios

albertoglezba commented on css/lagranfamilia.css:
```diff
@@ -480,6 +496,118 @@ body .ui-tooltip {
+	  }
+  }
+
+
```
¿El modal de bootstrap no se ve adecuado?

conaaxo replied:
se adecuan medias @media only screen and (max-width: 1199px) and (min-width: 601px){

y se reduce codigo

albertoglezba commented on css/memorama.css:
```diff
@@ -300,4 +300,10 @@ body {
+}
+
+
+footer{
+  position: fixed !important; /* AXO */
+  bottom: 0;
```
Igual que las anteriores

albertoglezba commented on css/rompecabezas.css:
```diff
@@ -213,4 +216,9 @@ body {
+
+footer{
+  position: fixed !important; /* AXO */
+  bottom: 0;
```
IDEM

albertoglezba commented on css/style.css:
```diff
@@ -422,13 +423,12 @@ footer{
 
 footer img{
-	margin: 21px 0px 0px 35px;
+	margin: 0px 0px 0px 0px;
```
usar clases auxiliares de bootstrap

albertoglezba commented on css/style.css:
```diff
@@ -541,3 +541,45 @@ footer p{
+	height: 50px;
+	clear: both;
+	width: 100%;
+}
```
Veo que el espaciados 75 es el más utilizado, ¿podríamos revisar estas reglas a ver cual se puede omitir?

albertoglezba commented on css/style.css:
```diff
@@ -541,3 +541,45 @@ footer p{
+
+
+/* 4X0 */
+body{
+	
+	min-height: 100vh;
+}
+footer{
+position: absolute;
+bottom: 0;
+}
```
Utilizar clases auxiliares de Bootstrap

albertoglezba commented on css/style_tablet.css:
Idealmente  habría que borrar este archivo y que el sitio sea responsivo, de no poderse, estos cambios se aplicar usando por las clases auxiliares de Bootstrap

albertoglezba commented on css/styles.css:
```diff
@@ -6554,14 +6554,14 @@ textarea.form-control-lg {
 }
 
-.placeholder {
+/*.placeholder {
   display: inline-block;
   min-height: 1em;
   vertical-align: middle;
   cursor: wait;
   background-color: currentColor;
   opacity: 0.5;
```
Borrar código comentado

conaaxo replied:
elimino comentarios

albertoglezba commented on css/styles.css:
Hay que revisar estas reglas, dudo que se utilicen las +6000 lineas de estilos (en todo caso, seccionarlas para archivos específicos).
Esto es un detalle que se viene arrastrando de versiones anteriores... podemos dejarlo para el mediano plazo.

albertoglezba commented on css/vecinos.css:
Procura no utilizar tu nombre en las clases que se utilizan

conaaxo replied:
los AXO los uso justo para pruebas, ya se eliminan los resagados

albertoglezba commented on invasoras/assets/animated-headline/README.md:
Archivo no necesario

albertoglezba commented on invasoras/assets/animated-headline/css/reset.css:
Es parte de un plugin, ¿es necesario? ¿no se puede realizar con lo ya existente?
La idea es aligerar el sitio, no engrosarlo más

conaaxo replied:
simplificando invasoras

albertoglezba commented on invasoras/assets/animated-headline/css/style.css:
Es parte de un plugin, ¿es necesario? ¿no se puede realizar con lo ya existente?
La idea es aligerar el sitio, no engrosarlo más

albertoglezba commented on invasoras/assets/animated-headline/index.html:
Archivo de ejemplo no es necesario añadirlo como commit para subir al repo o a producción

albertoglezba commented on invasoras/assets/animated-headline/js/jquery-2.1.1.js:
¿Es un jQuery modificado? ¿no se puede usar la versión que ya esta arriba? (En todo caso de no ser necesario el plugin, la biblioteca tampoco lo sería aquí)

albertoglezba commented on invasoras/assets/animated-headline/js/main.js:
Mismo comentario que arriba

albertoglezba commented on invasoras/assets/animated-headline/js/modernizr.js:
Acabo de darme cuenta que todo esto también se encuentra en raíz, mejor agarrarlo de ahí (sí es necesario)

conaaxo replied:
se fué

albertoglezba commented on invasoras/assets/animated-headline/partials/_layout.scss:
Acabo de darme cuenta que todo esto también se encuentra en raíz, mejor agarrarlo de ahí (sí es necesario)

albertoglezba commented on invasoras/assets/animated-headline/partials/_mixins.scss:
Acabo de darme cuenta que todo esto también se encuentra en raíz, mejor agarrarlo de ahí (sí es necesario)

albertoglezba commented on invasoras/assets/animated-headline/partials/_variables.scss:
Acabo de darme cuenta que todo esto también se encuentra en raíz, mejor agarrarlo de ahí (sí es necesario)

albertoglezba commented on invasoras/assets/animated-headline/scss/style.scss:
Acabo de darme cuenta que todo esto también se encuentra en raíz, mejor agarrarlo de ahí (sí es necesario)

albertoglezba commented on invasoras/assets/img/home/home/00.png:
Este archivo no se ocupa dentro de invasoras

albertoglezba commented on invasoras/assets/img/home/home/000.png:
Este archivo no se ocupa dentro de invasoras aparte de ser una imagen vacía

albertoglezba commented on invasoras/assets/img/home/home/03.jpg:
No encuentro referencia a esta imagen dentro de la carpeta de invasoras

albertoglezba commented on invasoras/assets/img/home/home/04.jpg:
No encuentro referencia a esta imagen dentro de la carpeta de invasoras

albertoglezba commented on invasoras/assets/img/home/home/12.jpg:
No encuentro referencia a esta imagen dentro de la carpeta de invasoras

albertoglezba commented on invasoras/assets/img/home/home/14.jpg:
No encuentro referencia a esta imagen dentro de la carpeta de invasoras

albertoglezba commented on invasoras/assets/img/home/home/21.jpg:
No encuentro referencia a esta imagen dentro de la carpeta de invasoras

albertoglezba commented on invasoras/assets/img/home/home/22.jpg:
No encuentro referencia a esta imagen dentro de la carpeta de invasoras

albertoglezba commented on invasoras/assets/img/home/home/23.jpg:
No encuentro referencia a esta imagen dentro de la carpeta de invasoras

albertoglezba commented on invasoras/assets/img/home/home/26.jpg:
No encuentro referencia a esta imagen dentro de la carpeta de invasoras

albertoglezba commented on invasoras/assets/img/home/home/CONABIO-logo2021-01.svg:
No encuentro referencia a esta imagen dentro de la carpeta de invasoras

albertoglezba commented on invasoras/assets/img/home/home/abajo.png:
No encuentro referencia a esta imagen dentro de la carpeta de invasoras

albertoglezba commented on invasoras/assets/img/home/home/guiamaestros.jpg:
No encuentro referencia a esta imagen dentro de la carpeta de invasoras

albertoglezba commented on invasoras/assets/img/home/home/guiamaestros_0.jpg:
No encuentro referencia a esta imagen dentro de la carpeta de invasoras

albertoglezba commented on invasoras/assets/img/home/home/logo-mexico-blanco.png:
No encuentro referencia a esta imagen dentro de la carpeta de invasoras

albertoglezba commented on invasoras/assets/img/home/home/logo58_conabio.png:
No encuentro referencia a esta imagen dentro de la carpeta de invasoras

albertoglezba commented on invasoras/assets/img/home/home/logo64_conabio.png:
No encuentro referencia a esta imagen dentro de la carpeta de invasoras

albertoglezba commented on invasoras/assets/img/home/home/logo_logob_gobmx.png:
No encuentro referencia a esta imagen dentro de la carpeta de invasoras

albertoglezba commented on invasoras/assets/img/home/CONABIO-logo2021-01.svg:
Este archivo no se ocupa dentro de invasoras

albertoglezba commented on invasoras/assets/img/home/home/logotipohome.png:
Este archivo no se utiliza dentro de invasoras.

albertoglezba commented on invasoras/assets/img/home/home/m_enlace.png:
No encuentro referencia a esta imagen dentro de la carpeta de invasoras

albertoglezba commented on invasoras/assets/img/home/home/maestros.png:
No encuentro referencia a esta imagen dentro de la carpeta de invasoras

albertoglezba commented on invasoras/assets/img/home/home/premio.png:
No encuentro referencia a esta imagen dentro de la carpeta de invasoras

albertoglezba commented on invasoras/assets/img/home/home/premio2.jpg:
No encuentro referencia a esta imagen dentro de la carpeta de invasoras

albertoglezba commented on invasoras/assets/img/home/home/premiounete.png:
No encuentro referencia a esta imagen dentro de la carpeta de invasoras

albertoglezba commented on invasoras/assets/img/home/home/programa.jpg:
No encuentro referencia a esta imagen dentro de la carpeta de invasoras

albertoglezba commented on invasoras/assets/img/home/home/tito.png:
No encuentro referencia a esta imagen dentro de la carpeta de invasoras

albertoglezba commented on invasoras/assets/img/home/home/titohome_1.png:
No encuentro referencia a esta imagen dentro de la carpeta de invasoras

albertoglezba commented on invasoras/assets/img/home/home/titologo.png:
No encuentro referencia a esta imagen dentro de la carpeta de invasoras

albertoglezba commented on invasoras/assets/img/home/home/titopapel.png:
No encuentro referencia a esta imagen dentro de la carpeta de invasoras

albertoglezba commented on invasoras/assets/mega-site-navigation/README.md:
Este archivo no es necesario subirlo ni a producción ni al repositorio

albertoglezba commented on invasoras/assets/mega-site-navigation/css/reset.css:
Al igual que animated-headline, este plugin se encuentra en raíz, ¿Por qué es necesario duplicarlo?

albertoglezba commented on invasoras/assets/mega-site-navigation/css/style.css:
Al igual que animated-headline, este plugin se encuentra en raíz, no creo conveniente duplicarlo.

albertoglezba commented on invasoras/assets/mega-site-navigation/img/cd-logo.svg:
Al igual que animated-headline, este plugin se encuentra en raíz, no creo conveniente duplicarlo.

albertoglezba commented on invasoras/assets/mega-site-navigation/img/img.jpg:
Al igual que animated-headline, este plugin se encuentra en raíz, no creo conveniente duplicarlo.

albertoglezba commented on invasoras/assets/mega-site-navigation/img/line-icon-1.svg:
Al igual que animated-headline, este plugin se encuentra en raíz, no creo conveniente duplicarlo.

albertoglezba commented on invasoras/assets/mega-site-navigation/img/line-icon-2.svg:
Al igual que animated-headline, este plugin se encuentra en raíz, no creo conveniente duplicarlo.

albertoglezba commented on invasoras/assets/mega-site-navigation/img/line-icon-3.svg:
Al igual que animated-headline, este plugin se encuentra en raíz, no creo conveniente duplicarlo.

albertoglezba commented on invasoras/assets/mega-site-navigation/img/line-icon-4.svg:
Al igual que animated-headline, este plugin se encuentra en raíz, no creo conveniente duplicarlo.

albertoglezba commented on invasoras/assets/mega-site-navigation/img/line-icon-5.svg:
Al igual que animated-headline, este plugin se encuentra en raíz, no creo conveniente duplicarlo.

albertoglezba commented on invasoras/assets/mega-site-navigation/img/line-icon-6.svg:
Al igual que animated-headline, este plugin se encuentra en raíz, no creo conveniente duplicarlo.

albertoglezba commented on invasoras/assets/mega-site-navigation/img/line-icon-7.svg:
Al igual que animated-headline, este plugin se encuentra en raíz, no creo conveniente duplicarlo.

albertoglezba commented on invasoras/assets/mega-site-navigation/img/line-icon-8.svg:
Al igual que animated-headline, este plugin se encuentra en raíz, no creo conveniente duplicarlo.

albertoglezba commented on invasoras/assets/mega-site-navigation/index.html:
Al igual que animated-headline, este plugin se encuentra en raíz, no creo conveniente duplicarlo.

albertoglezba commented on invasoras/assets/mega-site-navigation/js/jquery-2.1.1.js:
Al igual que animated-headline, este plugin se encuentra en raíz, no creo conveniente duplicarlo.

albertoglezba commented on invasoras/assets/mega-site-navigation/js/jquery.mobile.custom.min.js:
Al igual que animated-headline, este plugin se encuentra en raíz, no creo conveniente duplicarlo.

albertoglezba commented on invasoras/assets/mega-site-navigation/js/main.js:
Al igual que animated-headline, este plugin se encuentra en raíz, no creo conveniente duplicarlo.

albertoglezba commented on invasoras/assets/mega-site-navigation/js/modernizr.js:
Al igual que animated-headline, este plugin se encuentra en raíz, no creo conveniente duplicarlo.

albertoglezba commented on invasoras/assets/mega-site-navigation/partials/_layout.scss:
Al igual que animated-headline, este plugin se encuentra en raíz, no creo conveniente duplicarlo.

albertoglezba commented on invasoras/assets/mega-site-navigation/partials/_variables.scss:
Al igual que animated-headline, este plugin se encuentra en raíz, no creo conveniente duplicarlo.

albertoglezba commented on invasoras/assets/mega-site-navigation/scss/style.scss:
Al igual que animated-headline, este plugin se encuentra en raíz, no creo conveniente duplicarlo.

albertoglezba commented on invasoras/assets/morphing-modal/README.md:
Al igual que animated-headline, este plugin se encuentra en raíz, no creo conveniente duplicarlo.

albertoglezba commented on invasoras/assets/morphing-modal/css/reset.css:
Al igual que animated-headline, este plugin se encuentra en raíz, no creo conveniente duplicarlo.

albertoglezba commented on invasoras/assets/morphing-modal/css/style.css:
Al igual que animated-headline, este plugin se encuentra en raíz, no creo conveniente duplicarlo.

albertoglezba commented on invasoras/assets/morphing-modal/img/cd-icon-close.svg:
Al igual que animated-headline, este plugin se encuentra en raíz, no creo conveniente duplicarlo.

albertoglezba commented on invasoras/assets/morphing-modal/index.html:
Al igual que animated-headline, este plugin se encuentra en raíz, no creo conveniente duplicarlo.

albertoglezba commented on invasoras/assets/morphing-modal/js/jquery-2.1.1.js:
Al igual que animated-headline, este plugin se encuentra en raíz, no creo conveniente duplicarlo.

albertoglezba commented on invasoras/assets/morphing-modal/js/main.js:
Al igual que animated-headline, este plugin se encuentra en raíz, no creo conveniente duplicarlo.

albertoglezba commented on invasoras/assets/morphing-modal/js/modernizr.js:
Al igual que animated-headline, este plugin se encuentra en raíz, no creo conveniente duplicarlo.

albertoglezba commented on invasoras/assets/morphing-modal/js/velocity-LICENSE.md:
Al igual que animated-headline, este plugin se encuentra en raíz, no creo conveniente duplicarlo.

albertoglezba commented on invasoras/assets/morphing-modal/js/velocity.min.js:
Al igual que animated-headline, este plugin se encuentra en raíz, no creo conveniente duplicarlo.

albertoglezba commented on invasoras/assets/morphing-modal/partials/_layout.scss:
Al igual que animated-headline, este plugin se encuentra en raíz, no creo conveniente duplicarlo.

albertoglezba commented on invasoras/assets/morphing-modal/partials/_mixins.scss:
Al igual que animated-headline, este plugin se encuentra en raíz, no creo conveniente duplicarlo.

albertoglezba commented on invasoras/assets/morphing-modal/partials/_variables.scss:
Al igual que animated-headline, este plugin se encuentra en raíz, no creo conveniente duplicarlo.

albertoglezba commented on invasoras/assets/morphing-modal/scss/style.scss:
Al igual que animated-headline, este plugin se encuentra en raíz, no creo conveniente duplicarlo.

albertoglezba commented on invasoras/capsulas.html:
```diff
@@ -145,7 +104,8 @@ <h1><a href="index.html">Invasoras</a></h1>
 				<br><br>
```
Evitar espaciar con `br`

albertoglezba commented on invasoras/capsulas.html:
```diff
@@ -145,7 +104,8 @@ <h1><a href="index.html">Invasoras</a></h1>
 			   Con lo que has aprendido, mira a tu alrededor y trata de identificar alguna especie que no es parte de nuestra flora y fauna nativa.
-				</p> <br> <br>
+				</p>
+                <div class="spacer100"></div>
```
Entiendo esos div son para crear espacios... ¿Valen la pena? con las clases auxiliares de bootstrap no funcionarían?

albertoglezba commented on invasoras/css/capsulas.css:
No subir código comentado... eventualmente al igual que el resto de los css, hay que revisar que se usa, que no, y que se puede sustituir por clases auxiliares de bootstrap

albertoglezba commented on invasoras/css/invasion.css:
Hay que echarle un ojo a estas reglas, ya que el archivo de invasion.html el footer se come el contenido en pantallas pequeñas (HD 1366*768)
![Captura de pantalla 2023-05-02 a la(s) 22 18 25](https://user-images.githubusercontent.com/5861322/235832274-cd6553fc-8f5a-470f-9e47-dfdf51133314.png)

albertoglezba commented on invasoras/css/style.css:
```diff
@@ -401,23 +488,264 @@ footer hr{
+
+
+
+ 
```
en efecto, revisar

albertoglezba commented on invasoras/css/style_tablet.css:
Igual que los archivos anteriores de tableta, tratar de que quede con la responsividad de la biblioteca

albertoglezba commented on invasoras/invasion.html:
```diff
@@ -120,31 +89,77 @@ <h2>¡Acompáñame a conocer,<br>
+    <script src="js/jquery.sticky-kit.min.js"></script>
+    <script src="js/waypoints.min.js"></script>
+    <script src="js/jquery-ui.min.js"></script>
+    <script src="js/all.js"></script>
```
¿todo esto es necesario?

albertoglezba commented on invasoras/js/jquery-2.2.4.min.js:
¿Por qué se volvió a subir este archivo?

albertoglezba commented on js/encuentra.js:
¿Qué fue lo que cambió?

albertoglezba commented on js/jquery-3.6.4.min.js:
¿Por qué se volvió a subir este archivo?

albertoglezba commented on js/memorama.js:
¿Qué cambio aquí?

albertoglezba commented on lagranfamilia.html:
```diff
@@ -147,10 +147,10 @@ <h2 class="modalNombre">Jaguar</h2>
-
-
-    <footer class="footer bg-verdeselva small text-center text-white-50" id="footer"></footer>
+<div class="spacer150 colorlagranfamilia"></div>
```
Revisar espaciadores, utilizar clases de bootstrap

albertoglezba commented on mem_bosquet.html:
```diff
@@ -37,240 +61,109 @@
 
 </section>
-
+<div class="spacer75"></div>
```
idem

albertoglezba commented on mem_campo.html:
```diff
@@ -256,21 +167,40 @@ <h1><a href="index.html">M&eacute;xico el pa&iacute;s de las maravillas</a></h1>
 
 </section>
-
+<div class="spacer75"></div>
```
idem

albertoglezba commented on mem_campo.html:
```diff
@@ -256,21 +167,40 @@ <h1><a href="index.html">M&eacute;xico el pa&iacute;s de las maravillas</a></h1>
+<script src="assets/morphing-modal/js/modernizr.js"></script> <!-- Modernizr -->
 
+<!--  <script src="assets/full-screen-pushing-navigation-master/assets/js/util.js"></script> util functions included in the CodyHouse framework -->
+<script src="assets/mega-site-navigation/js/main.js"></script> <!-- mega menu -->
```
¡Esta duplicado este script, ya se esta agregando al momento de pegar el menú!
Agrego la lista de archivos donde potencialmente se tiene duplicidad:

![Captura de pantalla 2023-05-02 a la(s) 22 52 48](https://user-images.githubusercontent.com/5861322/235835156-ae6f81c1-80d7-4212-8c4f-fff253918fb0.png)

albertoglezba commented on arboles.html:
```diff
@@ -305,17 +209,35 @@ <h1><a href="index.html">M&eacute;xico el pa&iacute;s de las maravillas</a></h1>
+<!--scripts fuera del template base, tomé los del morphing modals, pero podrían ser en una carpeta afuera-->
+
+<!--  <script src="assets/full-screen-pushing-navigation-master/assets/js/util.js"></script> util functions included in the CodyHouse framework -->
+<script src="assets/mega-site-navigation/js/main.js"></script> <!-- mega menu -->
```
Plugin ya se esta insertando arriba en la inserción del menú

albertoglezba commented on rom_golfo.html:
```diff
@@ -276,21 +185,40 @@ <h1><a href="index.html">M&eacute;xico el pa&iacute;s de las maravillas</a></h1>
+<script src="assets/morphing-modal/js/modernizr.js"></script> <!-- Modernizr -->
+
+<!--  <script src="assets/full-screen-pushing-navigation-master/assets/js/util.js"></script> util functions included in the CodyHouse framework -->
+<script src="assets/mega-site-navigation/js/main.js"></script> <!-- mega menu -->
```
Plugin ya se esta insertando arriba en la inserción del menú

albertoglezba commented on rom_golfo.html:
```diff
@@ -276,21 +185,40 @@ <h1><a href="index.html">M&eacute;xico el pa&iacute;s de las maravillas</a></h1>
 
 </section>
-
+<div class="spacer75"></div>
```
Checar espaciadores

albertoglezba commented on color_bosquet.html:
```diff
@@ -242,21 +151,40 @@ <h1><a href="index.html">M&eacute;xico el pa&iacute;s de las maravillas</a></h1>
+<script src="assets/morphing-modal/js/modernizr.js"></script> <!-- Modernizr -->
+
+<!--  <script src="assets/full-screen-pushing-navigation-master/assets/js/util.js"></script> util functions included in the CodyHouse framework -->
+<script src="assets/mega-site-navigation/js/main.js"></script> <!-- mega menu -->
```
Plugin ya se esta insertando arriba en la inserción del menú

albertoglezba commented on color_golfo.html:
```diff
@@ -242,21 +151,40 @@ <h1><a href="index.html">M&eacute;xico el pa&iacute;s de las maravillas</a></h1>
+<script src="assets/morphing-modal/js/modernizr.js"></script> <!-- Modernizr -->
+
+<!--  <script src="assets/full-screen-pushing-navigation-master/assets/js/util.js"></script> util functions included in the CodyHouse framework -->
+<script src="assets/mega-site-navigation/js/main.js"></script> <!-- mega menu -->
```
Plugin ya se esta insertando arriba en la inserción del menú

albertoglezba commented on color_matorral.html:
```diff
@@ -242,21 +153,40 @@ <h1><a href="index.html">M&eacute;xico el pa&iacute;s de las maravillas</a></h1>
+<script src="assets/morphing-modal/js/modernizr.js"></script> <!-- Modernizr -->
 
+<!--  <script src="assets/full-screen-pushing-navigation-master/assets/js/util.js"></script> util functions included in the CodyHouse framework -->
+<script src="assets/mega-site-navigation/js/main.js"></script> <!-- mega menu -->
```
Plugin ya se esta insertando arriba en la inserción del menú

albertoglezba commented on color_pacifico.html:
```diff
@@ -242,19 +153,40 @@ <h1><a href="index.html">M&eacute;xico el pa&iacute;s de las maravillas</a></h1>
+<script src="assets/morphing-modal/js/modernizr.js"></script> <!-- Modernizr -->
+
+<!--  <script src="assets/full-screen-pushing-navigation-master/assets/js/util.js"></script> util functions included in the CodyHouse framework -->
+<script src="assets/mega-site-navigation/js/main.js"></script> <!-- mega menu -->
```
Plugin ya se esta insertando arriba en la inserción del menú

albertoglezba commented on color_pastizal.html:
```diff
@@ -242,19 +153,40 @@ <h1><a href="index.html">M&eacute;xico el pa&iacute;s de las maravillas</a></h1>
+<script src="assets/morphing-modal/js/modernizr.js"></script> <!-- Modernizr -->
+
+<!--  <script src="assets/full-screen-pushing-navigation-master/assets/js/util.js"></script> util functions included in the CodyHouse framework -->
+<script src="assets/mega-site-navigation/js/main.js"></script> <!-- mega menu -->
```
Plugin ya se esta insertando arriba en la inserción del menú

albertoglezba commented on color_selvah.html:
```diff
@@ -242,21 +153,40 @@ <h1><a href="index.html">M&eacute;xico el pa&iacute;s de las maravillas</a></h1>
+<script src="assets/morphing-modal/js/modernizr.js"></script> <!-- Modernizr -->
 
+<!--  <script src="assets/full-screen-pushing-navigation-master/assets/js/util.js"></script> util functions included in the CodyHouse framework -->
+<script src="assets/mega-site-navigation/js/main.js"></script> <!-- mega menu -->
```
Plugin ya se esta insertando arriba en la inserción del menú

albertoglezba commented on colorear.html:
```diff
@@ -0,0 +1,195 @@
+<script src="assets/morphing-modal/js/modernizr.js"></script> <!-- Modernizr -->
+
+<!--  <script src="assets/full-screen-pushing-navigation-master/assets/js/util.js"></script> util functions included in the CodyHouse framework -->
+<script src="assets/mega-site-navigation/js/main.js"></script> <!-- mega menu -->
```
Plugin ya se esta insertando arriba en la inserción del menú

albertoglezba commented on encuentra.html:
```diff
@@ -194,18 +114,40 @@ <h1><a href="index.html">M&eacute;xico el pa&iacute;s de las maravillas</a></h1>
+<script src="assets/morphing-modal/js/modernizr.js"></script> <!-- Modernizr -->
+
+<!--  <script src="assets/full-screen-pushing-navigation-master/assets/js/util.js"></script> util functions included in the CodyHouse framework -->
+<script src="assets/mega-site-navigation/js/main.js"></script> <!-- mega menu -->
```
Plugin ya se esta insertando arriba en la inserción del menú

albertoglezba commented on megadiverso.html:
```diff
@@ -488,18 +407,39 @@ <h1><a href="index.html">M&eacute;xico el pa&iacute;s de las maravillas</a></h1>
+<script src="assets/morphing-modal/js/modernizr.js"></script> <!-- Modernizr -->
+
+<!--  <script src="assets/full-screen-pushing-navigation-master/assets/js/util.js"></script> util functions included in the CodyHouse framework -->
+<script src="assets/mega-site-navigation/js/main.js"></script> <!-- mega menu -->
```
Plugin ya se esta insertando arriba en la inserción del menú

albertoglezba commented on mem_bosquet.html:
```diff
@@ -37,240 +61,109 @@
+<script src="assets/morphing-modal/js/modernizr.js"></script> <!-- Modernizr -->
 
+<!--  <script src="assets/full-screen-pushing-navigation-master/assets/js/util.js"></script> util functions included in the CodyHouse framework -->
+<script src="assets/mega-site-navigation/js/main.js"></script> <!-- mega menu -->
```
Plugin ya se esta insertando arriba en la inserción del menú

albertoglezba commented on mem_ciudad.html:
```diff
@@ -256,21 +166,40 @@ <h1><a href="index.html">M&eacute;xico el pa&iacute;s de las maravillas</a></h1>
+<script src="assets/morphing-modal/js/modernizr.js"></script> <!-- Modernizr -->
 
+<!--  <script src="assets/full-screen-pushing-navigation-master/assets/js/util.js"></script> util functions included in the CodyHouse framework -->
+<script src="assets/mega-site-navigation/js/main.js"></script> <!-- mega menu -->
```
Plugin ya se esta insertando arriba en la inserción del menú

albertoglezba commented on mem_golfo.html:
```diff
@@ -256,21 +166,40 @@ <h1><a href="index.html">M&eacute;xico el pa&iacute;s de las maravillas</a></h1>
+<script src="assets/morphing-modal/js/modernizr.js"></script> <!-- Modernizr -->
+
+<!--  <script src="assets/full-screen-pushing-navigation-master/assets/js/util.js"></script> util functions included in the CodyHouse framework -->
+<script src="assets/mega-site-navigation/js/main.js"></script> <!-- mega menu -->
```
Plugin ya se esta insertando arriba en la inserción del menú

albertoglezba commented on mem_golfo.html:
```diff
@@ -256,21 +166,40 @@ <h1><a href="index.html">M&eacute;xico el pa&iacute;s de las maravillas</a></h1>
 
 </section>
-
+<div class="spacer75"></div>
```
revisar espaciadores

albertoglezba commented on mem_matorral.html:
```diff
@@ -256,21 +167,40 @@ <h1><a href="index.html">M&eacute;xico el pa&iacute;s de las maravillas</a></h1>
+<script src="assets/morphing-modal/js/modernizr.js"></script> <!-- Modernizr -->
 
+<!--  <script src="assets/full-screen-pushing-navigation-master/assets/js/util.js"></script> util functions included in the CodyHouse framework -->
+<script src="assets/mega-site-navigation/js/main.js"></script> <!-- mega menu -->
```
Plugin ya se esta insertando arriba en la inserción del menú

albertoglezba commented on mem_pacifico.html:
```diff
@@ -256,21 +167,40 @@ <h1><a href="index.html">M&eacute;xico el pa&iacute;s de las maravillas</a></h1>
 
 </section>
-
+<div class="spacer75"></div>
```
revisar espaciadores

albertoglezba commented on mem_pacifico.html:
```diff
@@ -256,21 +167,40 @@ <h1><a href="index.html">M&eacute;xico el pa&iacute;s de las maravillas</a></h1>
+<script src="assets/morphing-modal/js/modernizr.js"></script> <!-- Modernizr -->
 
+<!--  <script src="assets/full-screen-pushing-navigation-master/assets/js/util.js"></script> util functions included in the CodyHouse framework -->
+<script src="assets/mega-site-navigation/js/main.js"></script> <!-- mega menu -->
```
Plugin ya se esta insertando arriba en la inserción del menú

albertoglezba commented on mem_pastizal.html:
```diff
@@ -256,21 +167,40 @@ <h1><a href="index.html">M&eacute;xico el pa&iacute;s de las maravillas</a></h1>
+<script src="assets/morphing-modal/js/modernizr.js"></script> <!-- Modernizr -->
 
+<!--  <script src="assets/full-screen-pushing-navigation-master/assets/js/util.js"></script> util functions included in the CodyHouse framework -->
+<script src="assets/mega-site-navigation/js/main.js"></script> <!-- mega menu -->
```
Plugin ya se esta insertando arriba en la inserción del menú

albertoglezba commented on mem_selvah.html:
```diff
@@ -37,240 +62,105 @@
+<script src="assets/morphing-modal/js/modernizr.js"></script> <!-- Modernizr -->
 
+<!--  <script src="assets/full-screen-pushing-navigation-master/assets/js/util.js"></script> util functions included in the CodyHouse framework -->
+<script src="assets/mega-site-navigation/js/main.js"></script> <!-- mega menu -->
```
Plugin ya se esta insertando arriba en la inserción del menú

albertoglezba commented on memoramas.html:
```diff
@@ -1,264 +1,149 @@
+<script src="assets/morphing-modal/js/modernizr.js"></script> <!-- Modernizr -->
 
+<!--  <script src="assets/full-screen-pushing-navigation-master/assets/js/util.js"></script> util functions included in the CodyHouse framework -->
+<script src="assets/mega-site-navigation/js/main.js"></script> <!-- mega menu -->
```
Plugin ya se esta insertando arriba en la inserción del menú

albertoglezba commented on obtenemos.html:
```diff
@@ -4,9 +4,12 @@
+    <meta name="description" content="México, país de las maravillas. Sitio web interactivo cuyo objetivo es el acercamiento de los niños al conocimiento de la riqueza natural del país a través de caricaturas, fotografías, videos, juegos y sonidos." />
+    <meta name="author" content="Comisión Nacional para el Conocimiento y Uso de la Biodiversidad, CONABIO." />
+
+    <title>¿Qué obtenemos? | el país de las maravillas</title>
```
"el" con "E" mayúscula

albertoglezba commented on publicaciones.html:
```diff
@@ -4,9 +4,12 @@
+    <meta name="description" content="México, país de las maravillas. Sitio web interactivo cuyo objetivo es el acercamiento de los niños al conocimiento de la riqueza natural del país a través de caricaturas, fotografías, videos, juegos y sonidos." />
+    <meta name="author" content="Comisión Nacional para el Conocimiento y Uso de la Biodiversidad, CONABIO." />
+
+    <title>Publicaciones | el país de las maravillas</title>
```
"el" con "E" mayúscula

albertoglezba commented on quedana.html:
```diff
@@ -4,9 +4,12 @@
+    <meta name="description" content="México, país de las maravillas. Sitio web interactivo cuyo objetivo es el acercamiento de los niños al conocimiento de la riqueza natural del país a través de caricaturas, fotografías, videos, juegos y sonidos." />
+    <meta name="author" content="Comisión Nacional para el Conocimiento y Uso de la Biodiversidad, CONABIO." />
+
+    <title>¿Cómo dañamos los ecosistemas? | el país de las maravillas</title>
```
"el" con "E" mayúscula

albertoglezba commented on quepasa.html:
```diff
@@ -4,9 +4,10 @@
+    <meta name="description" content="México, país de las maravillas. Sitio web interactivo cuyo objetivo es el acercamiento de los niños al conocimiento de la riqueza natural del país a través de caricaturas, fotografías, videos, juegos y sonidos." />
+    <meta name="author" content="Comisión Nacional para el Conocimiento y Uso de la Biodiversidad, CONABIO." />
+
+    <title>¿Qué pasa cuando eliminamos? | el país de las maravillas</title>
```
"el" con "E" mayúscula

albertoglezba commented on recortables.html:
```diff
@@ -4,9 +4,10 @@
+    <meta name="description" content="México, país de las maravillas. Sitio web interactivo cuyo objetivo es el acercamiento de los niños al conocimiento de la riqueza natural del país a través de caricaturas, fotografías, videos, juegos y sonidos." />
+    <meta name="author" content="Comisión Nacional para el Conocimiento y Uso de la Biodiversidad, CONABIO." />
+
+    <title>Recortables | el país de las maravillas</title>
```
"el" con "E" mayúscula

albertoglezba commented on rom_bosquet.html:
```diff
@@ -276,21 +184,40 @@ <h1><a href="index.html">M&eacute;xico el pa&iacute;s de las maravillas</a></h1>
+<script src="assets/morphing-modal/js/modernizr.js"></script> <!-- Modernizr -->
+
+<!--  <script src="assets/full-screen-pushing-navigation-master/assets/js/util.js"></script> util functions included in the CodyHouse framework -->
+<script src="assets/mega-site-navigation/js/main.js"></script> <!-- mega menu -->
```
Plugin ya se esta insertando arriba en la inserción del menú

albertoglezba commented on rom_campo.html:
```diff
@@ -276,21 +184,40 @@ <h1><a href="index.html">M&eacute;xico el pa&iacute;s de las maravillas</a></h1>
+<script src="assets/morphing-modal/js/modernizr.js"></script> <!-- Modernizr -->
+
+<!--  <script src="assets/full-screen-pushing-navigation-master/assets/js/util.js"></script> util functions included in the CodyHouse framework -->
+<script src="assets/mega-site-navigation/js/main.js"></script> <!-- mega menu -->
```
Plugin ya se esta insertando arriba en la inserción del menú

albertoglezba commented on rom_ciudad.html:
```diff
@@ -276,19 +184,40 @@ <h1><a href="index.html">M&eacute;xico el pa&iacute;s de las maravillas</a></h1>
+<script src="assets/morphing-modal/js/modernizr.js"></script> <!-- Modernizr -->
+
+<!--  <script src="assets/full-screen-pushing-navigation-master/assets/js/util.js"></script> util functions included in the CodyHouse framework -->
+<script src="assets/mega-site-navigation/js/main.js"></script> <!-- mega menu -->
```
Plugin ya se esta insertando arriba en la inserción del menú

albertoglezba commented on rom_matorral.html:
```diff
@@ -276,21 +184,40 @@ <h1><a href="index.html">M&eacute;xico el pa&iacute;s de las maravillas</a></h1>
+<script src="assets/morphing-modal/js/modernizr.js"></script> <!-- Modernizr -->
+
+<!--  <script src="assets/full-screen-pushing-navigation-master/assets/js/util.js"></script> util functions included in the CodyHouse framework -->
+<script src="assets/mega-site-navigation/js/main.js"></script> <!-- mega menu -->
```
Plugin ya se esta insertando arriba en la inserción del menú

albertoglezba commented on rom_pacifico.html:
```diff
@@ -276,21 +185,40 @@ <h1><a href="index.html">M&eacute;xico el pa&iacute;s de las maravillas</a></h1>
+<script src="assets/morphing-modal/js/modernizr.js"></script> <!-- Modernizr -->
+
+<!--  <script src="assets/full-screen-pushing-navigation-master/assets/js/util.js"></script> util functions included in the CodyHouse framework -->
+<script src="assets/mega-site-navigation/js/main.js"></script> <!-- mega menu -->
```
Plugin ya se esta insertando arriba en la inserción del menú

albertoglezba commented on rom_pastizal.html:
```diff
@@ -276,21 +183,40 @@ <h1><a href="index.html">M&eacute;xico el pa&iacute;s de las maravillas</a></h1>
+<script src="assets/morphing-modal/js/modernizr.js"></script> <!-- Modernizr -->
+
+<!--  <script src="assets/full-screen-pushing-navigation-master/assets/js/util.js"></script> util functions included in the CodyHouse framework -->
+<script src="assets/mega-site-navigation/js/main.js"></script> <!-- mega menu -->
```
Plugin ya se esta insertando arriba en la inserción del menú

albertoglezba commented on rom_selvah.html:
```diff
@@ -276,21 +183,40 @@ <h1><a href="index.html">M&eacute;xico el pa&iacute;s de las maravillas</a></h1>
+<script src="assets/morphing-modal/js/modernizr.js"></script> <!-- Modernizr -->
+
+<!--  <script src="assets/full-screen-pushing-navigation-master/assets/js/util.js"></script> util functions included in the CodyHouse framework -->
+<script src="assets/mega-site-navigation/js/main.js"></script> <!-- mega menu -->
```
Plugin ya se esta insertando arriba en la inserción del menú

albertoglezba commented on rompecabezas.html:
```diff
@@ -244,21 +156,40 @@ <h1><a href="index.html">M&eacute;xico el pa&iacute;s de las maravillas</a></h1>
+<script src="assets/morphing-modal/js/modernizr.js"></script> <!-- Modernizr -->
 
+<!--  <script src="assets/full-screen-pushing-navigation-master/assets/js/util.js"></script> util functions included in the CodyHouse framework -->
+<script src="assets/mega-site-navigation/js/main.js"></script> <!-- mega menu -->
```
Plugin ya se esta insertando arriba en la inserción del menú

conaaxo replied:
ok, se elimina de los que sea necesario

albertoglezba commented on sitiosweb.html:
```diff
@@ -4,9 +4,10 @@
+    <meta name="description" content="México, país de las maravillas. Sitio web interactivo cuyo objetivo es el acercamiento de los niños al conocimiento de la riqueza natural del país a través de caricaturas, fotografías, videos, juegos y sonidos." />
+    <meta name="author" content="Comisión Nacional para el Conocimiento y Uso de la Biodiversidad, CONABIO." />
+
+    <title>Sitios web | el país de las maravillas</title>
```
"el" con "E" mayúscula

albertoglezba commented on vecinos.html:
```diff
@@ -4,7 +4,10 @@
+    <meta name="description" content="México, país de las maravillas. Sitio web interactivo cuyo objetivo es el acercamiento de los niños al conocimiento de la riqueza natural del país a través de caricaturas, fotografías, videos, juegos y sonidos." />
+    <meta name="author" content="Comisión Nacional para el Conocimiento y Uso de la Biodiversidad, CONABIO." />
+
+    <title>Vecinos verdes | el país de las maravillas</title>
```
"el" con "E" mayúscula

albertoglezba commented on vecinos.html:
```diff
@@ -202,6 +206,14 @@ <h2>Sudamérica</h2>
+   
+    
+   
+     <section id="vecinosaxo"  data-color="#1D8895">
+        <img class="" src="assets/img/conoce/vecinosverde/bottom.png" alt="secday">
+    </section>
```
Evitar clases o ids personalizados, preferible que sean autoexplicativos.
Dicha imagen es un cuadro gris vacío, se puede sustituir con reglas css, o en su defecto, quitarse.

albertoglezba commented on invasoras/js/jquery-2.2.4.min.js:
¿Por qué se volvió a subir esta biblioteca?

albertoglezba reviewed (changes requested):
Hay varios archivos que no es necesario subirlos (imágenes, js, READMES, etc), podrías borrarlos.
Posteriormente atender los comentarios acerca de duplicación de inserciones de plugins.
Atender las peticiones de cambio de ser posible, y
ya con una lista más pequeña de archivos cambiados que se tenga la certeza que sí se tienen que subir, podríamos ver uno a uno como sustituir ciertas reglas de css.

conaaxo committed:
Comentarios

conaaxo committed:
Patrones a CSS

conaaxo committed:
INVASORAS RESPONSIVE

Se modificó estructura y estilos para comenzar a hacer responsiva la sección de INVASORAS

conaaxo committed:
Update rompecabezas.css

conaaxo committed:
INVASORAS

conaaxo committed:
INVASORAS

conaaxo committed:
INVASORAS

conaaxo committed:
INVASORAS
